Cargo.toml

```diff
@@ -80,12 +80,13 @@ mac-process-info = "0.2.0"
 osascript = "0.3.0"
 unix-daemonize = "0.1.2"
 
-[target."cfg(unix)"]
+#[target.'cfg(target_family = "unix")']
 
-[target."cfg(unix)".dependencies]
+[target.'cfg(target_family = "unix")'.dependencies]
 openssl-probe = "0.1.5"
 signal-hook = "0.3.14"
 crossbeam-channel = "0.5.5"
+sentry = { git = "https://github.com/getsentry/sentry-rust", version = "0.27.0", default-features = false, features = ["anyhow", "curl", "profiling", "frame-pointer"] }
```

Comment:
Specifying url should be unnecessary here.

Reply:
Without the `url` it's going to pull the latest published `crate` I think.

I'm not sure made it to a release yet, plus I'm about to merge a PR to align to a new standard format for profiles across all the SDKs so that is def. not in any release yet.

We can coordinate this with @Swatinem to make sure we make a release and at that point we can use a specific version instead of pulling from `master` ?

Reply:
https://github.com/getsentry/sentry-rust/pull/504 is already approved, so lets merge it, release new `sentry-rust` and replace it with regular cargo index as it was previously.

Reply:
I'll have to hold on for a bit for that.

I can merge that yet as the related [Relay](https://github.com/getsentry/relay/pull/1462), Sentry and Snuba PRs have yet to be merged.

Once those are merged I'll be able to merge the `sentry-rust`, than I can use that as you suggested.